main.py

```diff
@@ -0,0 +1,59 @@
+from fastapi import FastAPI, Depends, HTTPException
+import schemas
+import models
+import crud
+from sqlalchemy.orm import Session
+from database import SessionLocal, engine
+
+app = FastAPI()
+
+models.Base.metadata.create_all(bind=engine)
+
+
+@app.get("/")
+def read_root():
+    return {"message": "Hello, World!"}
+
+
+def get_db():
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+@app.get("/author/", response_model=list[schemas.AuthorRead])
+def read_authors(skip: int = 0,
+                 limit: int = 10, db: Session = Depends(get_db)):
+    return crud.get_all_authors(db=db, skip=skip, limit=limit)

```

Comment:
The `read_authors` function calls `crud.get_all_authors` with `skip` and `limit` parameters, but these parameters are not defined in the `get_all_authors` function in `crud.py`. You should add `skip` and `limit` parameters to the `get_all_authors` function to support pagination.